src/sentry/search/snuba/executors.py

```diff
@@ -1113,6 +1116,125 @@ class InvalidQueryForExecutor(Exception):
     pass
 
 
+def get_basic_group_snuba_lookup(search_filter: SearchFilter, attr_entity: Entity) -> Condition:
+    """
+    Returns the basic lookup for a search filter.
+    """
+    return Condition(
+        Column(f"group_{search_filter.key.name}", attr_entity),
+        Op.IN,
+        search_filter.value.raw_value,
+    )
+
+
+def get_assigned(search_filter: SearchFilter, attr_entity: Entity) -> Condition:
+    """
+    Returns the assigned lookup for a search filter.
+    """
+    users = filter(lambda x: isinstance(x, RpcUser), search_filter.value.raw_value)
+    user_ids = [user.id for user in users]
+    teams = filter(lambda x: isinstance(x, Team), search_filter.value.raw_value)
+    team_ids = [team.id for team in teams]
```

Comment:
nit: list comprehension are preferred over filter, and in this case can make it more compact and readable

```suggestion
    user_ids = [user.id for user in search_filter.value.raw_value if isinstance(user, RpcUser)]
    team_ids = [team.id for team in search_filter.value.raw_value if isinstance(team, Team)]
```